x-pack/plugins/siem/public/components/url_state/index.test.tsx

```diff
@@ -9,94 +9,118 @@ import toJson from 'enzyme-to-json';
 import * as React from 'react';
 import { Router } from 'react-router-dom';
 import { MockedProvider } from 'react-apollo/test-utils';
+import { StaticIndexPattern } from 'ui/index_patterns';
 
-import {
-  isKqlForRoute,
-  UrlStateContainer,
-  UrlStateContainerLifecycle,
-  UrlStateContainerLifecycleProps,
-} from './';
+import { UrlStateContainer, UrlStateContainerLifecycle } from './';
+import { LocationTypesNoNull, UrlStateContainerPropTypes } from './types';
+import { CONSTANTS } from './constants';
 import { apolloClientObservable, mockGlobalState, TestProviders } from '../../mock';
 import {
   createStore,
   hostsModel,
-  networkModel,
-  State,
   KueryFilterQuery,
+  networkModel,
   SerializedFilterQuery,
-  KueryFilterModel,
+  State,
 } from '../../store';
 import { ActionCreator } from 'typescript-fsa';
 import { InputsModelId } from '../../store/inputs/model';
 import { wait } from '../../lib/helpers';
 
 type Action = 'PUSH' | 'POP' | 'REPLACE';
 const pop: Action = 'POP';
-const location = {
+
+const filterQuery: KueryFilterQuery = {
+  expression: 'host.name:"siem-es"',
+  kind: 'kuery',
+};
+
+const serializedFilterQuery: SerializedFilterQuery = {
+  kuery: filterQuery,
+  serializedQuery:
+    '{"bool":{"should":[{"match_phrase":{"host.name":"siem-es"}}],"minimum_should_match":1}}',
+};
```

Comment:
Optional: You can easily copy paste this into a structure like this and it will make maintaining this easier down the road

```ts
  serializedQuery: JSON.stringify({
    bool: { should: [{ match_phrase: { 'host.name': 'siem-es' } }], minimum_should_match: 1 },
  }),
```